pkg/virt-launcher/virtwrap/util/libvirt_helper.go

```diff
@@ -375,15 +375,15 @@ func SetupLibvirt() (err error) {
 		if err != nil {
 			return err
 		}
-		err = os.Chmod("/dev/kvm", 0660)
+		err = os.Chmod("/dev/kvm", 0600)
```

Comment:
I'm not an expert here, is that a good idea? I read that in general the permissions should be 0660 for root user and kvm/qemu group?

Reply:
Will investigate these valuable comments. I will check over these specific statements for real minimal permissions.

Reply:
The virtual machine process is run under qemu user so you can't change the group permission to 0.

Reply:
@xpivarc I'm not sure I understand what you are saying. Do you imply that this change should be reverted?

Reply:
Thanks Daniel, this setting is reverted to 0660.